src/Microsoft.Azure.WebJobs.Host/Tables/StringToStorageTableConverter.cs

```diff
@@ -1,38 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using Microsoft.Azure.WebJobs.Host.Converters;
-using Microsoft.Azure.WebJobs.Host.Storage.Table;
-
-namespace Microsoft.Azure.WebJobs.Host.Tables
-{
-    internal class StringToStorageTableConverter : IConverter<string, IStorageTable>
```

Comment:
I don't have a good sense for the level of tests we had in place for all of these various binding permutations that are being removed. You're sure that we're covered for all of they (they continue to work?)